### DISCUSSION TITLE
Softlock fix: Remove tunnel block when exiting Pink Brinstar save room

### DISCUSSION THREAD

**Discussion by Nicholas-Steel** [tewtal/SMZ3Randomizer#151] [Ideas]
Remove block from tunnel to Pink Brinstar Save Room, when entering the room from the Save Room. This will resolve a soft-lock scenario without any invasive changes to level layout & has no effect on item requirements (other than removing the need to not run out of Power Bombs).

Image of location is available here:
https://discord.com/channels/567470487483973652/638112063780159499/792649643421990923
https://cdn.discordapp.com/attachments/638112063780159499/792649643175182336/unknown.png

Edit: or change the block to a Shot Block.

**Comment by TarThoron** [comment]
That's pink Brinstar, not Norfair.
